hugr-core/src/extension/resolution.rs

```diff
@@ -111,13 +103,14 @@ pub enum ExtensionResolutionError {
         available_extensions: Vec<ExtensionId>,
     },
     #[display(
-        "Type {ty} in {node} requires extension {missing_extension}, but it could not be found in the extension list used during resolution. The available extensions are: {}",
+        "Type {ty}{} requires extension {missing_extension}, but it could not be found in the extension list used during resolution. The available extensions are: {}",
+        node.map(|n| format!(" in {}", n)).unwrap_or_default(),
```

Comment:
what's the default behaviour?

Reply:
If there's a node: "Type Unit in Node(4) requires..."
If there's no node: "Type Unit requires ..."

Reply:
ah yeah sorry misread, thanks